test/utils.spec.js

```diff
@@ -506,6 +506,22 @@ describe('utils', () => {
       utils.safeExec('hello', myCb);
       expect(calls).to.deep.equal([[myErr, null]]);
     });
+
+    it('handles being unable to require `child_process`', () => {
+      utils._exec = null;
+
+      var actualErr = null;
+      var actualRes = null;
+      function myCb(err, res) {
+        actualErr = err;
+        actualRes = res;
+      }
+      utils.safeExec('hello', myCb);
+      expect(actualErr.toString()).to.equal(
+        new Error('exec not available').toString()
+      );
```

Comment:
I'd love if someone who is familiar with Node convention could check whether they like this. Basically I found that errors are not treated as equal by Chai even if they have the same error message, so fell back to this `toString()` solution, but there might be a better way.

Reply:
This works. We should upgrade our testing library to Jest.